src/test/regress/expected/multi_with_basics.out

```diff
@@ -0,0 +1,367 @@
+-- CTE cannot be outside of FROM/WHERE clause
+WITH cte AS (
```

Comment:
Well, it looks like we support this query if the subquery returns one rows. So, maybe use this example and update the comment? (I've no idea why people could use subqueries in the target list )
```SQL

 WITH cte AS (   SELECT user_id FROM users_table WHERE value_2 IN (1, 2) LIMIT 1  )
SELECT (SELECT * FROM cte);
```